tools/azure-sdk-tools/packaging_tools/sdk_package.py

```diff
@@ -39,6 +40,15 @@ def main(generate_input, generate_output):
         dist_path = Path(sdk_folder, folder_name, package_name, "dist")
         package["artifacts"] = [str(dist_path / package_file) for package_file in os.listdir(dist_path)]
         package["result"] = "succeeded"
+        # Generate api stub File
+        package_path = Path(sdk_folder, folder_name, package_name)
+        check_call(["python", "-m" "pip", "install", "-r", "../../../eng/apiview_reqs.txt",
+                    "--index-url=https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi"
+                    "/simple/"], cwd=package_path)
+        check_call(["apistubgen", "--pkg-path", "."], cwd=package_path)
+        for file in os.listdir(package_path):
+            if "_python.json" in file:
+                package["apiViewArtifact"] = str(Path(package_path, file))
```

Comment:
Can we use try-catch here? I think failure of creating ApiView shall not break whole pipeline. Maybe we can record `package["apiViewArtifact"] = False` and swagger UI show warning?